LICENSE

```diff
@@ -178,15 +179,15 @@ Apache License
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
it says to fill this in!

Reply:
no; it shows how you can apply the license by using this template and applying it to your file headers (and filling it in)

Reply:
https://github.com/moby/ipvs/blob/8f137da6850a975020f4f739c589d293dd3a9d7b/LICENSE#L180-L184

Reply:
So, if we want to do it fully correct;

- add a `NOTICE` file (containing the copyright)
- add a header to every file that has the boiler plating filled in with the copyright

Similar to https://github.com/docker/containerd-packaging/pull/136